libsolidity/interface/CompilerStack.h

```diff
@@ -239,7 +240,8 @@ class CompilerStack: boost::noncopyable
 		ContractDefinition const& _contract,
 		std::map<ContractDefinition const*, eth::Assembly const*>& _compiledContracts
 	);
-
+	/// Helper function to return path converted strings.
+	std::string sanitizePath(std::string const& _path) { return boost::filesystem::path(_path).generic_string(); }
```

Comment:
Can you move that closer to `absolutePath`? And it should be const end perhaps even static.